probeservices/checkin_test.go

```diff
@@ -0,0 +1,66 @@
+package probeservices_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/ooni/probe-engine/model"
+)
+
+func TestCheckInSuccess(t *testing.T) {
+	client := newclient()
+	client.BaseURL = "https://ams-pg-test.ooni.org"
+	config := model.CheckInConfig{
+		Charging:        true,
+		OnWiFi:          true,
+		Platform:        "string",
+		ProbeASN:        "string",
+		ProbeCC:         "string",
+		RunType:         "string",
+		SoftwareVersion: "string",
+		WebConnectivity: model.CategoryCodes{
+			CategoryCodes: []string{"NEWS", "CULTR"},
+		},
+	}
+	ctx := context.Background()
+	result, err := client.CheckIn(ctx, config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	//TODO what do we want to check?
```

Comment:
```suggestion
	// TODO(lorenzoPrimi): what do we want to check?
```

Reply:
Apart from this style-oriented comment, I think checking that the category codes are okay is already good. I also think we should be checking whether we have got a report ID.

Reply:
I don't understand this report_id for what is needed, and we are not gonna use it anytime soon. Do we need the test now?